app/models/katello/content_view.rb

```diff
@@ -621,6 +622,32 @@ def check_docker_repository_names!(environments)
       true
     end
 
+    def check_repositories_exist!
+      bad_repo = repositories.yum_type.find do |repo|
+        begin
+          ::Katello::Resources::Candlepin::Content.get(organization.label, repo.root.content_id)
```

Comment:
Since this is a backend candlepin API call, would it make sense to move this to the publish task instead?

Reply:
Moved it to the CV helper